DDL.sql

```diff
@@ -0,0 +1,263 @@
+CREATE TABLE bikeConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    distanceCovered INTEGER NOT NULL
+);
+
+CREATE TABLE parts(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    weight FLOAT NOT NULL
+);
+
+CREATE TABLE BMSConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    totalVoltageUnits TEXT DEFAULT NULL,
+    batteryVoltageUnits TEXT DEFAULT NULL,
+    currentUnits TEXT DEFAULT NULL,
+    thermistorTempUnits TEXT DEFAULT NULL,
+    packTempUnits TEXT DEFAULT NULL,
+    bqTempUnits TEXT DEFAULT NULL,
+    cellVoltageUnits TEXT DEFAULT NULL
+);
+
+CREATE TABLE IMUConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    vectorUnits TEXT DEFAULT NULL
+);
+
+CREATE TABLE TMUConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    thermUnits TEXT NOT NULL
+);
+
+CREATE TABLE TMSConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    sensorTempUnits TEXT NOT NULL,
+    pumpSpeedUnits TEXT NOT NULL,
+    fanSpeedUnits TEXT NOT NULL
+);
+
+CREATE TABLE PVCConfig(
+    ID SERIAL PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE BikeBoardConfig(
+    bikeConfigID INTEGER,
+    boardConfig INTEGER,
+    PRIMARY KEY ( bikeConfigID, boardConfig)
+);
+
+CREATE TABLE BikePartConfig(
+    ID SERIAL PRIMARY KEY,
+    bikeConfigId INTEGER NOT NULL,
+    partNumber TEXT NOT NULL
+);
+
+CREATE TABLE Context(
+    ID SERIAL PRIMARY KEY,
+    date DATE NOT NULL,
+    runEventType TEXT NOT NULL,
+    airTemp INTEGER DEFAULT NULL,
+    altitude INTEGER DEFAULT NULL,
+    location TEXT NOT NULL,
+    windVelocity FLOAT DEFAULT NULL,
+    riderOne TEXT NOT NULL,
+    riderTwo TEXT DEFAULT NULL,
+    riderOneCurrentWeight FLOAT NOT NULL,
+    riderTwoCurrentWeight FLOAT DEFAULT NULL,
+    humidity FLOAT DEFAULT NULL,
+    driverFeedback TEXT DEFAULT NULL
+);
+
+CREATE TABLE BMSNumAttemptsMade(
+    ID SERIAL PRIMARY KEY,
+    value INTEGER NOT NULL,
+    timeStamp TIMESTAMP,
+    context_id INTEGER REFERENCES Context
```

Comment:
Foreign key constraints shouldn't be done inline like this. Instead, put them on their own line as shown below:
```suggestion
    context_id INTEGER NOT NULL,
    FOREIGN KEY (context_id) REFERENCES Context(id)
```
Also, why is this lower_snake_case?

Reply:
Fixed. Out of curiosity why cant they be inline?